tests/python_package_test/test_dask.py

```diff
@@ -428,6 +382,7 @@ def test_ranker(output, client, listen_port, group):
     dask_ranker = dask_ranker.fit(dX, dy, sample_weight=dw, group=dg, client=client)
     rnkvec_dask = dask_ranker.predict(dX)
     rnkvec_dask = rnkvec_dask.compute()
+    rnkvec_local_1 = dask_ranker.to_local().predict(X)
```

Comment:
```suggestion
    rnkvec_dask_local = dask_ranker.to_local().predict(X)
```

Please change this name so it's clearer how it differs from `rnkvec_local`